aerogear/aerogear-unifiedpush-server
# Updating dependencies

## Motivation
Update upstream dependencies 

## What
This PR bumps many of the dependencies in Unified push to more decent versions.  Some dependencies had code changes and will be bumped at a later time.


## Verification Steps
This PR relies on a PR from aerogear-parent to be available.  
 
1. run MVN install on this PR https://github.com/aerogear/aerogear-parent/pull/86
2. Build UPS
3. Run UPS
4. Configure iOS and Android apps
4. Send a APNS Push Message
5. Send a Firebase Push Message
6. Verify messages were received on targets
7. Verify the node.js sender
8. Verify the Java sender

## Checklist:

- [ x\] Update AeroGear BOM
- [ x ] Update dependencies
- [  ] Update AeroGear bom to include the released version of the PR
- [  ] Update version information for release
## Progress

- [  ] Finished task
- [  ] Release on GitHub
- [  ] Release on DockerHub
